### PR TITLE
COMMERCE-11938

### PR DESCRIPTION
**Pull request by Brunno0liveira**
[COMMERCE-11938](https://liferay.atlassian.net/browse/COMMERCE-11938)

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceChannels.testcase [hunk 1]
```diff
@@ -967,6 +967,53 @@ definition {
...
+
+			Alert.viewSuccessMessage();
+
+			Click(locator1 = "Icon#CLOSE");
+
+			SelectFrameTop();
+
+			AssertVisible(
+				key_entryName = "Test Notification Template",
+				locator1 = "CommerceNavigation#ENTRY_NAME");
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceChannels.testcase:1013]: I think at the end of the test it's worth to check also that the email is correct inside the entry after saving the notification template

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by 22LucaGiannotta22** [comment]
Relevant failures are not related to the PR. This can be force-forwarded

**Comment by victoroscuro** [comment]
ci:forward:force

**Pull request closed without merging**